src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp

```diff
@@ -33,17 +34,15 @@ EndScriptData */
 #include "Player.h"
 #include "stratholme.h"
 
-enum Misc
-{
-    MAX_ENCOUNTER           = 6
-};
-
 enum InstanceEvents
 {
     EVENT_BARON_RUN         = 1,
     EVENT_SLAUGHTER_SQUARE  = 2
 };
 
+Position const timmyTheCruelSpawnPosition = { 3625.358f, -3188.108f, 130.3985f, 4.834562f };
+static AreaBoundary* const beforeScarletGate = new EllipseBoundary(Position(3671.158f, -3181.79f), 60.0f, 40.0f);
```

Comment:
is a static pointer initialized with "new" keyword a TC standard or something like that ? it looks quite strange to me, if possible I'd avoid a call to "new" here

Reply:
no need for static

Reply:
That's just copy & paste from this one https://github.com/TrinityCore/TrinityCore/blob/b453e124231a90321fe79fbf3a62acdcfa54a691/src/server/scripts/Northrend/Naxxramas/boss_gothik.cpp#L129 :P

Reply:
@Riztazz can you remove that static so it might be merged?